crunch/crunch.cpp

```diff
@@ -958,7 +958,7 @@ class crunch {
       else {
         console::info("CRN texture info:");
 
-        console::info("Width: %u, Height: %u, Levels: %u, Faces: %u\nBytes per block: %u, User0: 0x%08X, User1: 0x%08X, CRN Format: %u",
+        console::info("Width: %u, Height: %u, Levels: %u, Faces: %u\nBytes per block: %u, User0: 0x%08X, User1: 0x%08X, CRN Format: %l",

```

Comment:
This one doesn't quite fix the issue... apparently the crn_format enum, having both a -1 value and a 2^32 - 1 value, is a different size depending on the compiler. In MSVC it is 32 bits but in some others it must be 64. We could force it to be a specific type with the C++11 feature `enum crn_format: uint32_t` or whatever, instead of the old approach of trying to influence the type by adding fake values.